application/models/Add_model.php

```diff
@@ -0,0 +1,11 @@
+<?php
+class Add_model extends CI_Model 
```

Comment:
Use a model that represents the table in the database. Example: UserModel for users table and BooksModel for books table.